### PR TITLE
Math magicians: fetch data from API

### PR DESCRIPTION
**Pull request by noelfoka**
## Math magicians: fetch data from API

In this branch, I:

- [ ] Use the [API Ninjas](https://api-ninjas.com/api/quotes).
- [ ] Create a new component for displaying a quote.
- [ ] Fetch data inside of a new component.
- [ ] Show a `loading` state.
- [ ] Show an `error` state.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import { Component } from 'react';
 import Calculator from './components/Calculator';
+import Quote from './quote/Quotes';
 
 // eslint-disable-next-line react/prefer-stateless-function
 class App extends Component {
   render() {
     return (
```
Comment (V-Blaze) [inline src/App.js]: - You have done great so far, but there is some requirements you still need to meet before moving to the next milestone. :+1: :smile: 
- [ ] As required by Microverse, you are not meant to use any linters disable rule on your project. I'll suggest you remove the `disable rule` and fix the error. :+1: :100: Also I'll suggest you change this component to a `functional component` as that's part of the initial requirement and that's the current industrial standard. You can re-write the App component to:
 ` import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"
function App() {
  return (
    <>
      <Calculator />
      <Quotes />
    </>
  )
}
export default App`

- screenshot:
Screenshot from 2023-08-09 16-16-54

File: src/quote/Quotes.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react';
```
Comment (V-Blaze) [inline src/quote/Quotes.js]: > Create a new component for displaying a quote
- [ ] Following the best practices, this is a component t and it should be inside the component folder, you do not need to creat a new `quote folder` for this. All components are usually placed inside the `component folder` and that helps keep the project organized. :smile: :wave:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @noelfoka , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Quotes are fetched from API✔
-  Used useEffect correctly👋 
-  Quotes are displayed on the UI✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
Hi @noelfoka  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Quotes are fetched from API✔
Used useEffect correctly👋
Quotes are displayed on the UI✔
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**